src/components/proxy/QvProxyConfigurator.cpp

```diff
@@ -232,61 +232,101 @@ namespace Qv2ray::components::proxy
         QStringList actions;
         actions << QString("gsettings set org.gnome.system.proxy mode '%1'").arg("manual");
         bool isKDE = qEnvironmentVariable("XDG_SESSION_DESKTOP") == "KDE";
+        bool isDDE = isKDE ? false : qEnvironmentVariable("XDG_SESSION_DESKTOP") == "DDE";
```

Comment:
```suggestion
        bool isDDE = isKDE ? false : qEnvironmentVariable("XDG_SESSION_DESKTOP") == "deepin";
```
`XDG_SESSION_DESKTOP` is equal to `deepin` on DDE. You should consider to change it.